rtlib/include/rt/DisegniSegmenter.hpp

```diff
@@ -63,7 +79,10 @@ class DisegniSegmenter
     cv::Mat preprocess_();
 
     /** Run watershed on image */
-    static cv::Mat watershed_image_(const cv::Mat& input);
+    cv::Mat watershed_image_(const cv::Mat& input);
+
+    //Otsu Image Segmentation
+    cv::Mat otsu_segmentation_(const cv::Mat& input);
```

Comment:
Remove the Otsu stuff from this branch. We'll add it from the other branch if we keep it.